Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:16.04
+MAINTAINER Endless Services Team <services@endlessm.com>
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common
+
+LABEL version="0.1"
+
+RUN add-apt-repository ppa:alexlarsson/flatpak
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y apt-transport-https \
+                       curl \
+                       flatpak
+
+
+# Keep in line with requirements.txt and setup.py
```

Comment:
Would it be better to use pip to install the requirements? There would be no need to sync this manually with requirements.txt